src/index.mli

```diff
@@ -68,6 +68,9 @@ exception RO_not_allowed
 (** The exception raised when a write operation is attempted on a read_only
     index. *)
 
+exception Closed
+(** The exception raised when any operation is attempted on a closed index. *)
```

Comment:
any operation _except_ the `close` operation, which is a no-op on a closed index

Reply:
(perhaps this behaviour should be tested)

Reply:
Indeed.
I'm not sure how to test it, maybe by re-running the failing operations test on it?

Reply:
Done, thanks :heavy_check_mark: